compiler/rustc_symbol_mangling/src/typeid/typeid_itanium_cxx_abi.rs

```diff
@@ -548,6 +548,16 @@ fn encode_ty<'tcx>(
                 if let Some(value_str) = cfi_encoding.value_str() {
                     if !value_str.to_string().trim().is_empty() {
```

Comment:
Third one of `value_str.to_string().trim()` is too much, can be computed once?

Reply:
Done.